helm/config/nginx.conf

```diff
@@ -0,0 +1,38 @@
+#user  nginx;
+worker_processes  4;
+
+error_log  /var/log/nginx/error.log notice;
+pid        /tmp/nginx.pid;
+load_module modules/ngx_http_js_module.so;
+load_module /otel/otel_ngx_module.so;
+
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    opentelemetry_config /otel/otel-nginx.toml;
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    {{ if .Values.nginx.authorization.enabled }}
+    js_import auth from /etc/nginx/auth.js;
+    js_set $jwt_payload_sub auth.jwtPayloadSub;
+    {{ end }}
+
+    include /etc/nginx/log_format.conf;
+
+    access_log  /var/log/nginx/access.log main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    gzip  on;
```

Comment:
have you checked that it works?

Reply:
it seems to work after the check 
![image](https://github.com/MapColonies/geoserver-polygon-parts/assets/72082238/d02096a9-9394-4af7-bf3d-f1716e3ff3b3)